proposals/stack-switching/Explainer.md

```diff
@@ -833,6 +833,147 @@ events and only `(on $e switch)` handlers can handle `switch`
 events. The handler search continues past handlers for the wrong kind
 of event, even if they use the correct tag.
 
+#### Store extensions
+
+* New store component `conts` for allocated continuations
+  - `S ::= {..., conts <cont>?*}`
+
+* A continuation is a context annotated with its hole's arity
+  - `cont ::= (E : n)`
```

Comment:
Maybe bikeshedding: should this be called `continst` instead of `cont`? [Other components in the store are all named like fooinst](https://webassembly.github.io/spec/core/exec/runtime.html#syntax-store).